+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Tue Dec  6 23:50:46 2022 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL djvulibre
+ pkgs=djvulibre
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs djvulibre --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp2p6j3545
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp2p6j3545
+ cp ../common/test.pdf ./
+ expect
spawn any2djvu test.pdf
/-- Started Wed Dec  7 04:51:19 AM UTC 2022: root@anolis-riscv, pid 1038: /usr/bin/any2djvu (cwd /root/mugen-riscv/testcases/cli-test/djvulibre/oe_test_djvulibre_03)
WARNING!

any2djvu uses an external server which is willing to perform the
conversion and requires the document transfer over to that server.
There is a security issue in operating on documents not intended for
widespread distribution, which could be partially although not
completely ameliorated by using a secure web connection.

Do you acknowledge and allow the transmission of the document?
(Type 'yes' to acknowledge. You can define non-empty environment
 variable DJVU_ONLINE_ACK to avoid seeing this dialog, or use -a
 command line parameter to any2djvu).
[yes/no]:yes
Wed Dec 7 04:51:19 AM UTC 2022 Processing test ...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:06 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:07 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:08 --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:09 --:--:--     0curl: (6) Could not resolve host: any2djvu.djvu.org
error: something got wrong. check log file
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Tue Dec  6 23:51:30 2022 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ test -f test.djvu
+ CHECK_RESULT 1 0 0 'Check test.djvu not exist.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check test.djvu not exist.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check test.djvu not exist.'
+ LOG_ERROR 'Check test.djvu not exist.'
+ message='Check test.djvu not exist.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check test.djvu not exist.'
Tue Dec  6 23:51:31 2022 - ERROR - Check test.djvu not exist.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 37'
+ message='oe_test_djvulibre_03.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 37'
Tue Dec  6 23:51:31 2022 - ERROR - oe_test_djvulibre_03.sh line 37
+ return 0
+ ddjvu --help
+ grep 'Usage: ddjvu' result
Usage: ddjvu [options] [<djvufile> [<outputfile>]]
+ rm -rf result
+ CHECK_RESULT 0 0 0 'Check ddjvu --help failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu --help failed.'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ ddjvu -verbose test.djvu ddjvu1
ddjvu: [1-11711] Failed to open 'test.djvu': No such file or directory.
ddjvu: 'ByteStream.cpp:747'
ddjvu: Cannot open djvu document 'test.djvu'.
+ CHECK_RESULT 10 0 0 'Check ddjvu -verbose failed.'
+ actual_result=10
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu -verbose failed.'
+ '[' -z 10 ']'
+ '[' 0 -eq 0 ']'
+ test 10x '!=' 0x
+ test -n 'Check ddjvu -verbose failed.'
+ LOG_ERROR 'Check ddjvu -verbose failed.'
+ message='Check ddjvu -verbose failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu -verbose failed.'
Tue Dec  6 23:51:32 2022 - ERROR - Check ddjvu -verbose failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 42'
+ message='oe_test_djvulibre_03.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 42'
Tue Dec  6 23:51:33 2022 - ERROR - oe_test_djvulibre_03.sh line 42
+ return 0
+ test -f ddjvu1
+ CHECK_RESULT 1 0 0 'Check ddjvu1 not exist.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu1 not exist.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ddjvu1 not exist.'
+ LOG_ERROR 'Check ddjvu1 not exist.'
+ message='Check ddjvu1 not exist.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu1 not exist.'
Tue Dec  6 23:51:33 2022 - ERROR - Check ddjvu1 not exist.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 44'
+ message='oe_test_djvulibre_03.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 44'
Tue Dec  6 23:51:34 2022 - ERROR - oe_test_djvulibre_03.sh line 44
+ return 0
+ ddjvu -format=pgm test.djvu ddjvu1.pgm
ddjvu: [1-11711] Failed to open 'test.djvu': No such file or directory.
ddjvu: 'ByteStream.cpp:747'
ddjvu: Cannot open djvu document 'test.djvu'.
+ CHECK_RESULT 10 0 0 'Check ddjvu -format=pgm failed.'
+ actual_result=10
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu -format=pgm failed.'
+ '[' -z 10 ']'
+ '[' 0 -eq 0 ']'
+ test 10x '!=' 0x
+ test -n 'Check ddjvu -format=pgm failed.'
+ LOG_ERROR 'Check ddjvu -format=pgm failed.'
+ message='Check ddjvu -format=pgm failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu -format=pgm failed.'
Tue Dec  6 23:51:35 2022 - ERROR - Check ddjvu -format=pgm failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 46'
+ message='oe_test_djvulibre_03.sh line 46'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 46'
Tue Dec  6 23:51:35 2022 - ERROR - oe_test_djvulibre_03.sh line 46
+ return 0
+ test -f ddjvu1.pgm
+ CHECK_RESULT 1 0 0 'Check ddjvu1.pgm not exist.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu1.pgm not exist.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ddjvu1.pgm not exist.'
+ LOG_ERROR 'Check ddjvu1.pgm not exist.'
+ message='Check ddjvu1.pgm not exist.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu1.pgm not exist.'
Tue Dec  6 23:51:36 2022 - ERROR - Check ddjvu1.pgm not exist.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 48'
+ message='oe_test_djvulibre_03.sh line 48'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 48'
Tue Dec  6 23:51:36 2022 - ERROR - oe_test_djvulibre_03.sh line 48
+ return 0
+ ddjvu -aspect=no test.djvu ddjvu2
ddjvu: [1-11711] Failed to open 'test.djvu': No such file or directory.
ddjvu: 'ByteStream.cpp:747'
ddjvu: Cannot open djvu document 'test.djvu'.
+ CHECK_RESULT 10 0 0 'Check ddjvu -aspect=no failed.'
+ actual_result=10
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu -aspect=no failed.'
+ '[' -z 10 ']'
+ '[' 0 -eq 0 ']'
+ test 10x '!=' 0x
+ test -n 'Check ddjvu -aspect=no failed.'
+ LOG_ERROR 'Check ddjvu -aspect=no failed.'
+ message='Check ddjvu -aspect=no failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu -aspect=no failed.'
Tue Dec  6 23:51:37 2022 - ERROR - Check ddjvu -aspect=no failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 50'
+ message='oe_test_djvulibre_03.sh line 50'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 50'
Tue Dec  6 23:51:37 2022 - ERROR - oe_test_djvulibre_03.sh line 50
+ return 0
+ test -f ddjvu2
+ CHECK_RESULT 1 0 0 'Check ddjvu2 not exist.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu2 not exist.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ddjvu2 not exist.'
+ LOG_ERROR 'Check ddjvu2 not exist.'
+ message='Check ddjvu2 not exist.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu2 not exist.'
Tue Dec  6 23:51:38 2022 - ERROR - Check ddjvu2 not exist.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 52'
+ message='oe_test_djvulibre_03.sh line 52'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 52'
Tue Dec  6 23:51:38 2022 - ERROR - oe_test_djvulibre_03.sh line 52
+ return 0
+ ddjvu -mode=mask test.djvu ddjvu3
ddjvu: [1-11711] Failed to open 'test.djvu': No such file or directory.
ddjvu: 'ByteStream.cpp:747'
ddjvu: Cannot open djvu document 'test.djvu'.
+ CHECK_RESULT 10 0 0 'Check ddjvu -mode=mask failed.'
+ actual_result=10
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu -mode=mask failed.'
+ '[' -z 10 ']'
+ '[' 0 -eq 0 ']'
+ test 10x '!=' 0x
+ test -n 'Check ddjvu -mode=mask failed.'
+ LOG_ERROR 'Check ddjvu -mode=mask failed.'
+ message='Check ddjvu -mode=mask failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu -mode=mask failed.'
Tue Dec  6 23:51:39 2022 - ERROR - Check ddjvu -mode=mask failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 54'
+ message='oe_test_djvulibre_03.sh line 54'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 54'
Tue Dec  6 23:51:40 2022 - ERROR - oe_test_djvulibre_03.sh line 54
+ return 0
+ test -f ddjvu3
+ CHECK_RESULT 1 0 0 'Check ddjvu3 not exist.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu3 not exist.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ddjvu3 not exist.'
+ LOG_ERROR 'Check ddjvu3 not exist.'
+ message='Check ddjvu3 not exist.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu3 not exist.'
Tue Dec  6 23:51:40 2022 - ERROR - Check ddjvu3 not exist.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 56'
+ message='oe_test_djvulibre_03.sh line 56'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 56'
Tue Dec  6 23:51:41 2022 - ERROR - oe_test_djvulibre_03.sh line 56
+ return 0
+ ddjvu -mode=foreground test.djvu ddjvu4
ddjvu: [1-11711] Failed to open 'test.djvu': No such file or directory.
ddjvu: 'ByteStream.cpp:747'
ddjvu: Cannot open djvu document 'test.djvu'.
+ CHECK_RESULT 10 0 0 'Check ddjvu -mode=foreground failed.'
+ actual_result=10
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu -mode=foreground failed.'
+ '[' -z 10 ']'
+ '[' 0 -eq 0 ']'
+ test 10x '!=' 0x
+ test -n 'Check ddjvu -mode=foreground failed.'
+ LOG_ERROR 'Check ddjvu -mode=foreground failed.'
+ message='Check ddjvu -mode=foreground failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu -mode=foreground failed.'
Tue Dec  6 23:51:42 2022 - ERROR - Check ddjvu -mode=foreground failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 58'
+ message='oe_test_djvulibre_03.sh line 58'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 58'
Tue Dec  6 23:51:42 2022 - ERROR - oe_test_djvulibre_03.sh line 58
+ return 0
+ test -f ddjvu4
+ CHECK_RESULT 1 0 0 'Check ddjvu4 not exist.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu4 not exist.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ddjvu4 not exist.'
+ LOG_ERROR 'Check ddjvu4 not exist.'
+ message='Check ddjvu4 not exist.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu4 not exist.'
Tue Dec  6 23:51:43 2022 - ERROR - Check ddjvu4 not exist.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 60'
+ message='oe_test_djvulibre_03.sh line 60'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 60'
Tue Dec  6 23:51:43 2022 - ERROR - oe_test_djvulibre_03.sh line 60
+ return 0
+ ddjvu -mode=background test.djvu ddjvu5
ddjvu: [1-11711] Failed to open 'test.djvu': No such file or directory.
ddjvu: 'ByteStream.cpp:747'
ddjvu: Cannot open djvu document 'test.djvu'.
+ CHECK_RESULT 10 0 0 'Check ddjvu -mode=background failed.'
+ actual_result=10
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu -mode=background failed.'
+ '[' -z 10 ']'
+ '[' 0 -eq 0 ']'
+ test 10x '!=' 0x
+ test -n 'Check ddjvu -mode=background failed.'
+ LOG_ERROR 'Check ddjvu -mode=background failed.'
+ message='Check ddjvu -mode=background failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu -mode=background failed.'
Tue Dec  6 23:51:44 2022 - ERROR - Check ddjvu -mode=background failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 62'
+ message='oe_test_djvulibre_03.sh line 62'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 62'
Tue Dec  6 23:51:45 2022 - ERROR - oe_test_djvulibre_03.sh line 62
+ return 0
+ test -f ddjvu5
+ CHECK_RESULT 1 0 0 'Check ddjvu5 not exist.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu5 not exist.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ddjvu5 not exist.'
+ LOG_ERROR 'Check ddjvu5 not exist.'
+ message='Check ddjvu5 not exist.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu5 not exist.'
Tue Dec  6 23:51:45 2022 - ERROR - Check ddjvu5 not exist.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 64'
+ message='oe_test_djvulibre_03.sh line 64'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 64'
Tue Dec  6 23:51:46 2022 - ERROR - oe_test_djvulibre_03.sh line 64
+ return 0
+ ddjvu -skip test.djvu ddjvu6
ddjvu: [1-11711] Failed to open 'test.djvu': No such file or directory.
ddjvu: 'ByteStream.cpp:747'
ddjvu: Cannot open djvu document 'test.djvu'.
+ CHECK_RESULT 10 0 0 'Check ddjvu -skip failed.'
+ actual_result=10
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu -skip failed.'
+ '[' -z 10 ']'
+ '[' 0 -eq 0 ']'
+ test 10x '!=' 0x
+ test -n 'Check ddjvu -skip failed.'
+ LOG_ERROR 'Check ddjvu -skip failed.'
+ message='Check ddjvu -skip failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu -skip failed.'
Tue Dec  6 23:51:46 2022 - ERROR - Check ddjvu -skip failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 66'
+ message='oe_test_djvulibre_03.sh line 66'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 66'
Tue Dec  6 23:51:47 2022 - ERROR - oe_test_djvulibre_03.sh line 66
+ return 0
+ test -f ddjvu6
+ CHECK_RESULT 1 0 0 'Check ddjvu6 not exist.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ddjvu6 not exist.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ddjvu6 not exist.'
+ LOG_ERROR 'Check ddjvu6 not exist.'
+ message='Check ddjvu6 not exist.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check ddjvu6 not exist.'
Tue Dec  6 23:51:47 2022 - ERROR - Check ddjvu6 not exist.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 68'
+ message='oe_test_djvulibre_03.sh line 68'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 68'
Tue Dec  6 23:51:48 2022 - ERROR - oe_test_djvulibre_03.sh line 68
+ return 0
+ djvm --help
+ grep Usage: result
Usage:
+ rm -rf result
+ CHECK_RESULT 0 0 0 'Check djvm --help failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check djvm --help failed.'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ djvmcvt --help
+ grep Usage: result
Usage:
+ rm -rf result
+ CHECK_RESULT 0 0 0 'Check djvmcvt --help failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check djvmcvt --help failed.'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ djvmcvt -b test.djvu djvmcvt.djvu
DjVuDocument.fail_URL	file://localhost/root/mugen-riscv/testcases/cli-test/djvulibre/oe_test_djvulibre_03/test.djvu
+ CHECK_RESULT 1 0 0 'Check djvmcvt -b failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check djvmcvt -b failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check djvmcvt -b failed.'
+ LOG_ERROR 'Check djvmcvt -b failed.'
+ message='Check djvmcvt -b failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check djvmcvt -b failed.'
Tue Dec  6 23:51:49 2022 - ERROR - Check djvmcvt -b failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 76'
+ message='oe_test_djvulibre_03.sh line 76'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 76'
Tue Dec  6 23:51:50 2022 - ERROR - oe_test_djvulibre_03.sh line 76
+ return 0
+ test -f djvmcvt.djvu
+ CHECK_RESULT 1 0 0 'Check djvmcvt.djvu not exist.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check djvmcvt.djvu not exist.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check djvmcvt.djvu not exist.'
+ LOG_ERROR 'Check djvmcvt.djvu not exist.'
+ message='Check djvmcvt.djvu not exist.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check djvmcvt.djvu not exist.'
Tue Dec  6 23:51:50 2022 - ERROR - Check djvmcvt.djvu not exist.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 78'
+ message='oe_test_djvulibre_03.sh line 78'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 78'
Tue Dec  6 23:51:51 2022 - ERROR - oe_test_djvulibre_03.sh line 78
+ return 0
+ djvmcvt -i test.djvu ../oe_test_djvulibre_03/ djvmcvt1.djvu
DjVuDocument.fail_URL	file://localhost/root/mugen-riscv/testcases/cli-test/djvulibre/oe_test_djvulibre_03/test.djvu
+ CHECK_RESULT 1 0 0 'Check djvmcvt -i failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check djvmcvt -i failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check djvmcvt -i failed.'
+ LOG_ERROR 'Check djvmcvt -i failed.'
+ message='Check djvmcvt -i failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check djvmcvt -i failed.'
Tue Dec  6 23:51:51 2022 - ERROR - Check djvmcvt -i failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 80'
+ message='oe_test_djvulibre_03.sh line 80'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 80'
Tue Dec  6 23:51:52 2022 - ERROR - oe_test_djvulibre_03.sh line 80
+ return 0
+ test -f ./djvmcvt1.djvu
+ CHECK_RESULT 1 0 0 'Check djvmcvt1.djvu not exist.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check djvmcvt1.djvu not exist.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check djvmcvt1.djvu not exist.'
+ LOG_ERROR 'Check djvmcvt1.djvu not exist.'
+ message='Check djvmcvt1.djvu not exist.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check djvmcvt1.djvu not exist.'
Tue Dec  6 23:51:52 2022 - ERROR - Check djvmcvt1.djvu not exist.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_djvulibre_03.sh line 82'
+ message='oe_test_djvulibre_03.sh line 82'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_djvulibre_03.sh line 82'
Tue Dec  6 23:51:53 2022 - ERROR - oe_test_djvulibre_03.sh line 82
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Tue Dec  6 23:51:53 2022 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 19
+ test 19 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Dec  6 23:51:54 2022 - ERROR - failed to execute the case.
+ exit 19
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Tue Dec  6 23:51:54 2022 - INFO  - Start to restore the test environment.
++ ls
++ grep -v '\.sh'
+ rm -rf test-any2djvu.log test.pdf
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp2p6j3545 ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp2p6j3545
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Tue Dec  6 23:52:06 2022 - INFO  - End to restore the test environment.
